en/components/drop_down_virtual.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Drop Down Component
+_description: Use Ignite UI for Angular Virtualized Drop Down to display a very large list of items which supports a single item selection.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Drop Down components, Angular Drop Down controls, Angular Control Large Item List, Angular Control Singe Selection
+---
+
+## Virtual Drop Down
+
+<p class="highlight">[The Ignite UI for Angular Drop Down](drop_down.md) can fully integrate usage of the [IgxForOf directive](for_of.md) in order to display a very large list of items for its selection</p>
+
+<div class="sample-container loading" style="height:220px">
+    <iframe id="dropdown-virtual-iframe" src='{environment:demosBaseUrl}/data-entries/dropdown-virtual' width="100%" height="100%" seamless="" frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="dropdown-virtual-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+## Configuration
+
+In order for the drop down to display a list of virtual items, some prerequisites need to be fulfilled.
```

Comment:
Let's reword a little bit to -> In order to configure the drop down to display a list of virtual items, you need to fulfil some prerequisites

Reply:
Addressed in the latest commit